ee/codegen/src/__test__/project.test.ts

```diff
@@ -4777,4 +4777,148 @@ baz = foo + bar
       }
     });
   });
+
+  describe("module data with additional files", () => {
+    const displayData = {
+      workflow_raw_data: {
+        nodes: [
+          {
+            id: "entry",
+            type: "ENTRYPOINT",
+            data: {
+              label: "Entrypoint",
+              source_handle_id: "entry_source",
+              target_handle_id: "entry_target",
+            },
+            inputs: [],
+          },
+          {
+            id: "templating-node",
+            type: "TEMPLATING",
```

Comment:
In general, we should prefer our generic node's in tests instead of TEMPLATING nodes going forward